tests/README.md

```diff
@@ -69,6 +69,21 @@ supported by minikube).
 You can also use any existing Kubernetes cluster, as long as you can access it
 with kubectl from the same host you are running the tests on.
 
+For example, to run the tests with a local minikube environment instead of the
+default minikube docker-in-docker container, execute:
+```sh
+$ cd <root_of_cloned_repo>
+$ minikube start --memory=4096 --extra-config=kubelet.authorization-mode=AlwaysAllow --extra-config=kubelet.anonymous-auth=true
```

Comment:
This is pretty convoluted.

Reply:
Removed the kubelet auth options from the doc and minikube startup.